<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findDialog</class>
 <widget class="QDialog" name="findDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>102</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="vLayout1">
     <item>
      <layout class="QHBoxLayout" name="hLayout1">
       <item>
        <widget class="QLabel" name="findWhatLabel">
         <property name="text">
          <string>Find What</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="keyWordsLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gLayout1">
       <item row="0" column="0">
        <widget class="QCheckBox" name="matchCaseLineEdit">
         <property name="text">
          <string>Match Case</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="findNextButton">
         <property name="text">
          <string>Find Next</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="findPreviousButton">
         <property name="text">
          <string>Find Previous</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>findDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
